README.adoc

```diff
@@ -141,25 +147,30 @@ The `@Timed` annotation applies to the `get()` method to track how frequently th
 invoked and also how long it takes for the invocation to complete. This annotation is
 followed by fields that correspond to metadata fields:
 
-* The `name` field is optional and sets the name of the metric.
-* The `unit` field has a default value of `seconds`.
-* The `description` field is optional and describes the purpose of the metric
-annotation.
+|===
+| *Field* | *Purpose*
+|`name` | Optional and sets the name of the metric.
+|`description` | Optional and describes the purpose of the metric annotation.
+|===
 
 The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
 you request the list of systems methods by visiting the `http://localhost:9080/inventory/systems` URL to
 retrieve the information for a list of all previously registered hosts:
 
-* The `absolute` field is set to `true` to give the metric the name of the method (without the class and package names).
-* The `description` field is optional and describes the purpose of the metric annotation.
-* The `monotonic` field is set to `true` to make the counter increment only.
+|===
+| *Field* | *Purpose*
+| `absolute` | Gives the metric the name of the method without the class and package names if it is set to `true`.
+| `monotonic` | Makes the counter increment only if it is set to `true`.
+|===
 
 The `@Gauge` annotation on the `getTotal()` method either tracks the inventory size or the number of systems that are stored
 in the inventory. A system is stored in the inventory when you visit the
 `http://localhost:9080/inventory/systems/localhost` URL to retrieve the information for a specific host:
 
-* Specify the `unit` field. This annotation has no default value.
-* The `name` and `description` fields are optional.
+|===
+| *Field* | *Purpose*
+| `unit` | Sets the unit of the metric.
```

Comment:
How would you handle `name` and `description`?

Reply:
I assume that there is no need to explain them again.

Reply:
I am not sure I understand.  You removed the line ... and we have no need to explain them again?

Reply:
No. the `name` and `description` field were repeated again in `@Gauge` annotation. So There is repeat their definitions again since it is already explained in the first table

Reply:
If we only have 1 or few entries each, I am not sure if we need to use tables.  Or, we may not need the header row.  Please see what is best/appropriate from a presentation standpoint, @Ahmad-Ayyoub @evelinec.  Thanks.

Reply:
Since it is for a different annotation, I am not sure if we can simply drop it and assume users can tie the 2 together.  But, I don't have the full doc in view so can't comment further.